applications/notes/pages/home.html

```diff
@@ -3,31 +3,41 @@
 
 <head>
   <title>Sign in- Notedly</title>
-  <link href="styles.css" rel="stylesheet" />
+  <link href="../styles.css" rel="stylesheet" />
 </head>
 
 <body>
   <div class="container">
+    <span class="page-header">
+      <img src="../imgs/favicon.ico" alt="">
+      <h1 class="">Notedly</h1>
```

Comment:
You shouldn't have a block element like `<h1>` inside of an inline element like `<span>`. I suggest changing the `<span>` to a `<header>`.

Reply:
OOps! True, I don't know why I did that!  Could've used a div even or header which clearly it is!
Will change that . Thanks!